composer.json

```diff
@@ -21,14 +21,18 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
-        "guzzle/guzzle": "~3.8",
+        "php": ">=5.4.8",
         "symfony/event-dispatcher": "~2.3",
-        "symfony/options-resolver": "~2.3"
+        "symfony/options-resolver": "~2.3",
+        "psr/http-message-implementation": "~1.0",
+        "php-http/adapter-implementation": "^0.1.0" ,
+        "php-http/discovery": "^0.1@dev",
+        "php-http/message-decorator": "^0.1.0"
```

Comment:
Got rid of the Guzzle dependency. We now only have generic and virtual dependencies.